libs/ui/dataviz/src/lib/chart/chart.component.ts

```diff
@@ -27,6 +27,7 @@ import {
   ScatterController,
   Tooltip,
 } from 'chart.js'
+import { truncateString } from '../utils/utils'
```

Comment:
I don't think this is really necessary to introduce a new function... this util does exactly the same thing as `String.substring()`

Reply:
I guess you are right, you'd just truncate without ending the string with `...` then?

Reply:
oh right sorry, I overlooked the '...' added at the end! still, I think adding a new `utils/utils.ts` file is a bit awkward, I mean this function could just be part of `chart.component.ts` maybe? just to remove a bit of verbosity

Reply:
the thing is it's quite specific to chart.js because usually this would be handled in CSS

Reply:
Ok, I think I got your point. I added a commit to move it to the component. Just wondering if handling the scope in the callback could be done in a more elegant way (to access both the function and the parent scope).